src/System.Security.Principal.Windows/src/System/Security/Principal/Win32.cs

```diff
@@ -132,7 +132,7 @@ internal static byte[] ConvertIntPtrSidToByteArraySid(IntPtr binaryForm)
 
             try
             {
-                if (TRUE != Interop.mincore.ConvertStringSidToSid(stringSid, out ByteArray))
```

Comment:
`findstr` says that the only two hits in this library are the definition and this usage.  So now it's a dead const, and I agree, it should be deleted with this change.